chainercv/experimental/links/model/fcis/utils/mask_voting.py

```diff
@@ -40,7 +40,7 @@ def _mask_aggregation(
 
 
 def mask_voting(
-        roi_cmask_prob, bbox, roi_cls_prob, size,
```

Comment:
How about `roi_seg_prob` --> `seg_prob` and `roi_cls_prob` -> `cls_prob`.

Also, there is an inconsistency in the documentation and the variable names.